src/ExposureHistory/ExposureDetail.tsx

```diff
@@ -22,10 +23,26 @@ const ExposureDetail: FunctionComponent = () => {
   useStatusBarEffect("light-content")
   const { t } = useTranslation()
 
+  const [connectivity, setConnectivity] = useState<boolean | null | undefined>(
+    true,
+  )
+
   const {
     GAEN_AUTHORITY_NAME: healthAuthorityName,
     AUTHORITY_ADVICE_URL: healthAuthorityLink,
   } = env
+
+  useEffect(() => {
+    const unsubscribe = NetInfo.addEventListener((state) => {
+      // netInfo state comes as null while unresolved so to avoid flicker we only set component state
+      // if the netInfo state is resolved to boolean
+      if (state.isInternetReachable !== null) {
```

Comment:
Not sure but could this use `state.isInternetReachable && setConnectivity(state.isInternetReachable)`?

Reply:
We actually need to specifically check for null here because that is this method uninitialized state. Also its a bit more readable this way so I think it should be fine.